hypothesis-python/src/hypothesis/internal/conjecture/dfa/__init__.py

```diff
@@ -477,3 +617,18 @@ def transition(self, state, char):
                     if u <= char <= v:
                         return j
             return DEAD
+
+    def raw_transitions(self, i):
+        if i == DEAD:
+            return
+        transitions = self.__transitions[i]
+        if isinstance(transitions, dict):
+            yield from sorted(transitions.items())
```

Comment:
Is it worth caching `sorted(transitions.items())` so that we don't have to repeatedly sort it?

Reply:
Hmm. I'm not sure. Probably not? It's always a relatively small list with cheap comparisons.

It would probably be worth going over all this code with a profiler at some point, but honestly I haven't yet and have only done very crude benchmarking, so I'm not sure it's worth making that sort of change yet.